index.html

```diff
@@ -2,6 +2,8 @@
 <html>
 	<head>
 		<title>PakistanJS</title>
+		<meta name="keywords" content="pakistan js javascript nodejs node angular react rxjs code live-coding pakistan.js pakistanjs.com">
+		<meta name="description" content="A website for people who want to become software engineers focused. Providing Urdu/ Hindi content for learning Javascript and other technologes">
```

Comment:
maybe, change this to something along these lines

`A community effort to help nurture JavaScript talent in Pakistan.` ?